.github/workflows/price-feeder-release.yml

```diff
@@ -19,6 +19,9 @@ jobs:
           go-version: 1.19
           cache: true
           cache-dependency-path: price-feeder/go.sum
+          flavor: |
```

Comment:
We need to add this one inside the `price-feeder-docker.yml` file at docker metadata

Reply:
thanks for catching. Updated. I'm testing in a local fork and will report if everything works.